components/NewOutivityForm/index.js

```diff
@@ -0,0 +1,155 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function NewOutivityForm({
+  CreateOutivity,
+  handleCancel,
+  selectedImage,
```

Comment:
this should be written in camelCase, and I don't think you will need the selectedImage:

```suggestion
  createOutivity,
  handleCancel,
```